src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext } from "vscode-azureextensionui";
```

Comment:
You should put most of the files in a folder called `src/commands/downloadProject/`

As mentioned, "download" is more appropriate than "clone" and since Docker is not the main focus of this feature we don't need a specific folder for "dockersupport"

Reply:
I also renamed cloneLocally.ts to downloadProject.ts and moved it to the downloadProject folder